scripts/build/utils.mjs

```diff
@@ -52,8 +52,6 @@ export async function exec(cmd, args, options = {}) {
         }
         else {
             proc.unref();
-            // wait a short period in order to allow the process to start successfully before Node exits.
-            setTimeout(() => resolve({ exitCode: undefined }), 100);

```

Comment:
Doesn't someone still need to resolve this promise? Otherwise hereby will just keep running? (I haven't tested quite yet)

Reply:
Ah, fair point. I'll fix.